Snippets/AppContext/01-app-context.swift

```diff
@@ -0,0 +1,272 @@
+  metadataId: "some-id",
+  name: "Some User",
+  custom: ["department": "Engineering"]
+)
+
+pubnub.setUserMetadata(userMetadataToSet) { result in
+  switch result {
+  case let .success(userMetadata):
+    print("The metadata for `\(userMetadata.metadataId)`: \(userMetadata)")
+  case let .failure(error):
+    print("Create request failed with error: \(error.localizedDescription)")
+  }
+}
+// snippet.end
+
+// snippet.remove-user-metadata
+pubnub.removeUserMetadata("some-id") { result in
+  switch result {
+  case let .success(metadataId):
+    print("The metadata has been removed for the identifier `\(metadataId)`")
+  case let .failure(error):
+    print("Delete request failed with error: \(error.localizedDescription)")
+  }
+}
+// snippet.end
+
+// snippet.all-channel-metadata
+pubnub.allChannelMetadata { result in
+  switch result {
+  case let .success(response):
+    print("The channel metadata objects \(response.channels)")
+    print("The next page used for pagination: \(String(describing: response.next))")
+  case let .failure(error):
+    print("Fetch All request failed with error: \(error.localizedDescription)")
+  }
+}
+// snippet.end
+
+// snippet.fetch-channel-metadata
+pubnub.fetchChannelMetadata("some-channel-id") { result in
+  switch result {
+  case let .success(channelMetadata):
+    print("The metadata for `\(channelMetadata.metadataId)`: \(channelMetadata)")
+  case let .failure(error):
+    print("Fetch request failed with error: \(error.localizedDescription)")
+  }
+}
+// snippet.end
+
+// snippet.set-channel-metadata
+let channelMetadataToSet = PubNubChannelMetadataBase(
+  metadataId: "some-channel-id",
+  name: "Channel Name"
+)
+
+pubnub.setChannelMetadata(channelMetadataToSet) { result in
+  switch result {
+  case let .success(channelMetadata):
+    print("The metadata for `\(channelMetadata.metadataId)`: \(channelMetadata)")
+  case let .failure(error):
+    print("Create request failed with error: \(error.localizedDescription)")
+  }
+}
+// snippet.end
+
+// snippet.update-existing-channel-metadata
+// Simulates a reference to already fetched channel metadata
+var existingChannelMetadata = PubNubChannelMetadataBase(
+  metadataId: "yourChannelMetadataId",
+  name: "Offtopic",
+  type: "Miscellaneous",
+  status: "active",
+  custom: ["admin": "John Stones"]
+)
+
+// Modify properties directly, such as updating `custom` and `name`
+existingChannelMetadata.custom = ["admin": "Charles Dawson"]
+existingChannelMetadata.name = "Unrelated"
+
+// When you're ready, set the updated metadata on the server to persist the changes:
+pubnub.setChannelMetadata(existingChannelMetadata) {
+  switch $0 {
+  case let .success(metadata):
+    print("Did successfully set channel metadata with \(metadata.metadataId)")
+  case let .failure(error):
+    print("Unexpected error along the way: \(error)")
+  }
+}
+// snippet.end
+
+// snippet.remove-channel-metadata
+pubnub.removeChannelMetadata("some-channel-id") { result in
```

Comment:
I've noticed, that description for methods disappeared starting from here. In previous file before call and `snippet` start there were short message about what we are doing.